test/gold_cc_input_test.dart

```diff
@@ -0,0 +1,98 @@
+@TestOn('browser')
+library polymer_elements.test.gold_cc_input;
+
+import 'package:polymer_elements/gold_cc_input.dart';
+import 'package:polymer_elements/iron_input.dart';
+import 'package:web_components/web_components.dart';
+import 'package:test/test.dart';
+import 'common.dart';
+
+main() async {
+  await initWebComponents();
+
+  group('basic', () {
+    test('input is spaced out correctly', () {
+      GoldCcInput input = fixture('basic');
+      input.value = '12345678';
+      expect(input.value, equals('1234 5678'));
+    });
+
+    test('invalid input is not ok', () {
+      GoldCcInput input = fixture('required');
+      input.value = '1234';
+      forceXIfStamp(input);
+
+      var container = input.querySelector('paper-input-container');
+      expect(container, isNotNull);
+      expect(container.invalid, isTrue);
+      expect(input.cardType, equals(''));
+    });
+
+    test('valid input is ok', () {
+      GoldCcInput input = fixture('required');
+      input.value = '4000000000000002';
+      forceXIfStamp(input);
+
+      var container = input.querySelector('paper-input-container');
+      expect(container, isNotNull);
+      expect(container.invalid, isFalse);
+      expect(input.cardType, equals('visa'));
+    });
+
+    test('empty required input shows error', () {
+      GoldCcInput input = fixture('required');
+      forceXIfStamp(input);
+
+      var error = input.querySelector('paper-input-error');
+      expect(error, isNotNull);
+      expect(error.getComputedStyle().display, isNot('none'),
+          reason: 'error should not be display:none');
+    });
+
+    test('invalid input shows error message after manual validation', () {
+      GoldCcInput input = fixture('ErrorWithoutAutoValidate');
+      forceXIfStamp(input);
+
+      var error = input.querySelector('paper-input-error');
+      expect(error, isNotNull);
+
+      // The error message is only displayed after manual validation.
+      expect(error.getComputedStyle().display, equals('none'),
+          reason: 'error should be display:none');
+      input.validate();
+      expect(error.getComputedStyle().display, isNot('none'),
+          reason: 'error should not be display:none');
+    }, skip: 'https://github.com/dart-lang/polymer_elements/issues/3');
```

Comment:
Lets create a new issue for this one as well, or possibly one issue for all skipped gold-element tests.